docs/endpoints/configs/stampede3.yaml

```diff
@@ -0,0 +1,41 @@
+display_name: Stampede3@TACC
+
+engine:
+  type: GlobusComputeEngine
+  max_workers_per_node: 2
+  worker_debug: False
+
```

Comment:
Is there a value to "cluttering" the configuration with the default value of `worker_debug`?

Reply:
There isn't, I've removed this one.